src/bundle/Resources/views/themes/admin/content/tab/policies/tab.html.twig

```diff
@@ -1,10 +1,72 @@
 {% trans_default_domain 'policy' %}
 {% import _self as tab %}
-{% include '@ezdesign/ui/table_header.html.twig' with { header_text: 'tab.policies.policies'|trans|desc('Policies') } %}
 
-{{  include('@ezdesign/content/tab/policies/table.html.twig', {
-        'policies_pager': policies_pager,
-}) }}
+{% set body_rows = [] %}
+{% if policies_pager.currentPageResults is not empty %}
+    {% for policy in policies_pager.currentPageResults %}
+        {% set body_row_cols = [] %}
+
+        {% set col_raw %}
+            <a href="{{ path('ezplatform.role.view', { roleId: policy.role_assignment.role.id}) }}">
+                {{ policy.role_assignment.role.identifier }}
+            </a>
+        {% endset %}
+        {% set body_row_cols = body_row_cols|merge([{
+            content: col_raw,
+        }]) %}
+
+        {% set col_raw %}
+            {%- if policy.role_assignment.usergroup is defined -%}
+                {{ ez_content_name( policy.role_assignment.usergroup ) }}
+            {%- else -%}
+                {{ ez_content_name( policy.role_assignment.user ) }}
+            {%- endif -%}
+        {% endset %}
+        {% set body_row_cols = body_row_cols|merge([{
+            content: col_raw,
+        }]) %}
+
+        {% set body_row_cols = body_row_cols|merge([
+            { content: ('role.policy.' ~ policy.module)|trans({}, 'forms') },
+            { content: policy.function|capitalize },
+        ]) %}
+
+        {% set col_raw %}
+            {%- if policy.limitations is not empty -%}
+                <ul class="list-unstyled">
+                    {%- for limitation in policy.limitations -%}
+                        <li>
+                            <span class="font-weight-bold" title="{{ 'policy.limitation.identifier_tooltip'|trans({'%identifier%': limitation.identifier})|desc('"%identifier%" Limitation') }}">
```

Comment:
Personally I don't like this modifier, but I guess it's ok after all...

Reply:
I just moved it from the other place in this file...